ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/aaa-server-groups.yml

```diff
@@ -0,0 +1,45 @@
+---
+## AAA Server Groups
+aaa_server_groups:
+  - name: TACACS
```

Comment:
The `type` key should be `required` in schema.
Also, the same issue as tacacs server -  `servers` key is not required but its not checked in template so it throws error if not defined.